src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx

```diff
@@ -113,48 +132,140 @@ To chat with New Relic AI about log errors:
+## Fill alerting gaps [#coverage-gaps]
+
+As your team and systems evolve, staying ahead of potential issues is crucial. Alerts are your early warning system, but keeping them up to date can be challenging. New Relic AI can help identify areas where you might be missing critical notifications.
+
+Chat with New Relic AI and ask:
 
+```
+Show me all of my entities that are not covered by an alert. 
 ```
 
-## Secure data
+```
+Show me my OpenTelemetry services that are not covered by an alert.
+```
+
+## Create synthetic monitors [#synthetic]
 
-Data security and privacy are primary concerns for anyone building and maintaining software. We want you to know that anything you ask our AI is safe. NRAI is built with the same security controls and standards that safeguard New Relic customer data. We will never share your data with other accounts or customers.  We work with vendors that meet our privacy and security standards, and we carefully assess prospective vendors in these critical areas before engaging their services. 
+Synthetic monitoring helps you proactively catch and resolve issues before they affect your customers. You can now create a synthetic monitor by chatting with our AI. Using New Relic AI, create a ping check monitor from the default public location (Portland, Oregon) and check the availability of a URL at your desired cadence. 
 
-Our AI uses a variety of inputs to improve its responses and always provides the best answer. For example, it uses inputs like our product documentation, a large library of example queries, and your telemetry data. But even though our AI scans your data as needed to provide context and help you build charts and understand response times, that data is _never stored_ by New Relic.
+For example:
 
-## Always learning 
+```
+Create a synthetic monitor to check the URL 'newrelic.com' every 10 minutes and name it: 'my awesome monitor'.
+```
 
-New Relic AI is becoming more accurate and responsive the more it's used. We want to build the AI experience you want. When chatting with our AI, please give its response a thumbs up or a thumbs down.  If you notice something that isn't right or that could be improved, let us know. Go to the user menu, click <DoNotTranslate>**Help**</DoNotTranslate>, then click <DoNotTranslate>**Give us feedback**</DoNotTranslate>.
+To create a ping check monitor with NRAI:
+
+    1. Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities)** **All Entities**.
+    2. Click **Ask AI** in the top right-hand corner.
+    3. Tell AI to create a ping check monitor based on your parameters.
+    4. Click **Create monitor**.
+    5. If you want to see your settings or make any changes, select **Go to monitor**.
+    6. Select ** Detailed configuration ** if you want more advanced settings or access to other types of synthetic monitors.
+
+## Check the status of support tickets [#support]
```

Comment:
@homelessbirds can I do this from anywhere in the UI?

Reply:
Yes, you just need to ask ai.

Reply:
Which is why I have the section called "Chat with AI" at the top, so whenever I say "Chat with AI" below, you can refer back to the section about how to chat with AI.